cypress/support/fragments/requests/requests.js

```diff
@@ -229,4 +241,50 @@ export default {
     }).as('getTags');
     cy.wait('@getTags');
   },
+
+  requestTypes: { PAGE: 'Page', HOLD: 'Hold', RECALL: 'Recall' },
+
+  selectHoldsRequestType() {
+    cy.do(Checkbox({ name: 'Hold' }).click());
+  },
+
+  selectPagesRequestType() {
+    cy.do(Checkbox({ name: 'Page' }).click());
+  },
+
+  selectRecallsRequestType() {
+    cy.do(Checkbox({ name: 'Recall' }).click());
+  },
+
+  verifyFilteredResults(requestType) {
+    const values = [];
+    cy.get('[data-row-index]').each($row => {
+      cy.get(`[class*="mclCell-"]:nth-child(${5})`, { withinSubject: $row })
```

Comment:
try to use id's and attributes instead of nth-child

Reply:
@dmitriyfolio Unfortunately, rows doesn't have special `class` names only have `grid-cell` class and only way to extract some cell value is to know its order in the list